config/pop-os/24.04.mk

```diff
@@ -58,7 +58,7 @@ POST_DISTRO_PKGS+=systemd-boot
 
 ifeq ($(NVIDIA),1)
 DISTRO_PARAMS+=modules_load=nvidia
-DISTRO_PARAMS+=nvidia-drm.modeset=0
+DISTRO_PARAMS+=nvidia-drm.modeset=1
```

Comment:
This is an override, so can it just be removed to restore the default?

Reply:
I'm uncertain. I think we need it explicitly in the kernel parameters due to sequencing related to https://github.com/pop-os/linux/pull/332#

But, I do know it is needed for the ISO to return start a GUI session. As we move out of alpha, we'll probably want to shake the config and drop anything unnecessary. In the short term, I know this will achieve our goal.

Reply:
I assume `modeset=1` is still being set by system76-power and included in initramfs? If so, then it depends on if `parse_early_param` works with those modprobe options instead of kernel command line.

Reply:
That I do not know. It's out of my scope of knowledge. If you have a more elegant solution, feel free to run with it.